.github/workflows/VerifyAppChanges.yaml

```
@@ -0,0 +1,21 @@
+name: 'Verify App Changes'
+
+on:
+  pull_request:
+    branches: [ 'main', 'releases/*']
+
+concurrency:
+  group: ${{ github.workflow }}-${{ github.event.pull_request.number }}
+  cancel-in-progress: true
+
+jobs:
+  VerifyAppChanges:
+    runs-on: windows-latest
+    name: Verify App Changes
+    steps:
+      - name: checkout
+        uses: actions/checkout@v4
```

Use a pinned depedency :) 
```suggestion
        uses: actions/checkout@692973e3d937129bcbf40652eb9f2f61becf3332 # v4.1.7
```